cmd/node/config.go

```diff
@@ -22,4 +22,16 @@ var customBuckets = map[string][]float64{
 		// 1ms, 2ms, 5ms, 10ms, 25ms, 50ms, 100ms, 250ms, 500ms, 1s, 2.5s, 5s, 10s, 20s
 		0.001, 0.002, 0.005, 0.01, 0.025, 0.05, 0.1, 0.25, 0.5, 1, 2.5, 5, 10, 20,
 	},
+	"keydb_keys_hashing_duration_seconds": {
+		// 1microsecond, 2.5microsecond, 5microsecond, 1ms, 5ms, 10ms, 25ms, 50ms, 100ms, 250ms, 500ms, 1s
+		0.00001, 0.00025, 0.0005, 0.001, 0.005, 0.01, 0.025, 0.05, 0.1, 0.25, 0.5, 1,
```

Comment:
Do we really care about the 1, 2.5 and 5 microseconds? It seems a bit too much. Maybe just 100µs should be a good enough bucket for the µsecs group?

Reply:
Let's keep up to 10ms :+1: